packager/dockerpublisher.go

```diff
@@ -0,0 +1,52 @@
+package packager
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/suborbital/subo/project"
+	"github.com/suborbital/subo/subo/util"
+)
+
+const (
+	dockerPublishJobType = "docker"
+)
+
+type DockerPublishJob struct{}
+
+// NewDockerPublishJob returns a new PublishJob for Bindle.
+func NewDockerPublishJob() PublishJob {
```

Comment:
Comment references Bindle, that should be Docker.

Also the constructor might not be needed here, `new(DockerPublishJob)` will achieve the same thing.

Reply:
Good call, also I love 'real' constructors :)